chpl/chpl-resources/src/main/resources/environment.properties.template

```diff
@@ -106,9 +106,12 @@ asyncCacheRefreshEnabled=true
 #emailBuilder.config.productionEnvironment=false
 #Let emails being sent to this list of domains go through
 #emailBuilder.config.whitelistedDomains=ainq.com,onc.gov
+#Value is appended to the end subject, can be used to indicate what environment sent an email
+#emailBuilder.config.emailSubjectSuffix=[Local]
```

Comment:
This line seems weird...

Reply:
Sorry, the description is fine. Why is there a commented out version of the config variable when the actual config variable is 3 or 4 lines down?